corehq/apps/users/bulk_download.py

```diff
@@ -227,7 +299,7 @@ def _dump_xlsx_and_expose_download(filename, headers, rows, download_id, task, t
     DownloadBase.set_progress(task, total_count, total_count)
 
 
-def dump_users_and_groups(domain, download_id, user_filters, task, owner_id):
+def dump_users_and_groups(domain, download_id, user_filters, task, is_web_download, owner_id):
```

Comment:
It looks like the web user download will include a tab for groups, is that right? I don't think it should, since groups only relate to mobile users.